ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2400,8 +2400,41 @@ tests:
       TEST_ADDITIONAL: Network_Observability
     test:
     - chain: openshift-e2e-test-qe-destructive
-    workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
-- as: baremetalds-ipi-ovn-ipv4-dualstack-v6-primary-f4
+    workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
+- as: baremetalds-ipi-ovn-ipv6-fips-f14
+  cron: 37 13 2,6,10,14,19,24,28 * *
```

Comment:
The cron need to be regenerated, you can use `ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh` to generate an uniq cron

Reply:
done